indico/web/client/styles/base/_layout.scss

```diff
@@ -9,6 +9,14 @@ body {
     @extend .flexcol;
 }
 
+/* Work arond issues with Firefox which only displays the first page when
+ * printing */
```

Comment:
single line with `//` is fine (or wrap before `*/`)

Reply:
I wanted to keep the same style as the comment for the WebKit fix below and not exceed 80 chars per line. See new commit to keep style consistent.